src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -525,11 +525,11 @@
           - az application-gateway waf-config list-rule-sets
```

Comment:
I just noticed that these populator commands are wrong.  They should be `az network application-gateway ...`. They are missing the "network" bit. 😮